src/init.ts

```diff
@@ -48,12 +48,25 @@ function startLater(
  */
 function start(settings: ClientSettings): Promise<SessionResult> {
     const activeProviders = settings.providers.filter((p) => p.shouldRun())
-    return Promise.allSettled(
-        activeProviders.map((provider) => provider.fetchSessionConfig()),
+    // We can't use Promise.allSettled() below due to browser support.
+    // Therefore, we implement it manually via p.then().catch() to collect all
+    // results regardless of their resolution status. I.e. we polyfill.
+    return Promise.all(
+        activeProviders.map((provider) => provider.fetchSessionConfig())
+        .map((p) =>
+            p.then((value) => ({
+                status: "fulfilled",
+                value
+            }))
+            .catch((reason) => ({
+                status: 'rejected',
```

Comment:
Does Prettier catch these single-quotes?

Reply:
@phamann I added a PR on the Fastly fork with a possible fix for this: https://github.com/fastly/openinsights/pull/9

Reply:
Thanks!